src/components/common/FullWalletListRow.js

```diff
@@ -143,6 +145,7 @@ class FullWalletListRowLoadedComponent extends Component<FullWalletListRowLoaded
             <View style={[styles.rowContent]}>
               <View style={styles.rowIconWrap}>
                 {symbolImageDarkMono && <Image style={[styles.rowCurrencyLogoAndroid]} source={{ uri: symbolImageDarkMono }} resizeMode="cover" />}
+                <Pie size={styles.rowCurrencyOverlaySize} color={'rgba(0, 0, 0, 0.5)'} progress={this.getProgress()} style={styles.rowCurrencyOverlayLogo} />
```

Comment:
@paullinator will talk to @amanalili about re-writing this component directly in our app, since we had to modify the upstream to get what we wanted anyhow. In the mean time, this works.